RevenueCatUI/CustomerCenter/Views/PurchaseCardView.swift

```diff
@@ -78,13 +75,7 @@ struct PurchaseInformationCardView: View {
             self.subtitle = purchaseInformation.pricePaidString(localizations: localization)
         }
 
-        if let refundStatus, let message = refundStatus.subtitle(localization: localization) {
-            self.additionalInfoTitle = localization[.refundStatus]
-            self.additionalInfoSubtitle = message
-        } else {
-            self.additionalInfoTitle = nil
-            self.additionalInfoSubtitle = nil
-        }
+        self.additionalInfoSubtitle = refundStatus?.subtitle(localization: localization)
```

Comment:
Nit: since the old title was removed, perhaps this could be renamed to `additionalInfoTitle`. Or even better, just `additionalInfo`